assignment1/src/com/assignment1/InputOutput.java

```diff
@@ -0,0 +1,96 @@
+package com.assignment1;
+import java.util.Scanner;
+
+public class InputOutput {
+	Scanner s = new Scanner(System.in);
+	
+	public Item inputItem() {
+		Item item = new Item();
+		//s.nextLine();
+		System.out.println("Please Enter the details of the item:-");
+		System.out.print("-name : ");
+		item.setItemName(s.nextLine());
+		System.out.print("-price : ");
+		item.setItemPrice(s.nextDouble());
+		System.out.print("-quantity : ");
+		item.setItemQuantity(s.nextInt());
+		System.out.print("-type : ");
+		item.setItemType(s.next());
+		
+		return item;
+	}
+	
+	
+	public Item calculateTax(Item item) throws InvalidTypeException {
+		String type = item.getItemType();
+		double tax;
+		switch(type) {
```

Comment:
The switch case can be avoided by using different type of items. The various item type could have been implemented as different implementations of an abstract Item class.

Reply:
Okay